### ISSUE TITLE
【EESS】New submissions for Fri, 28 Jul 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#473]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Understanding Silent Failures in Medical Image Classification
 - **Authors:** Till J. Bungert, Levin Kobelke, Paul F. Jaeger
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2307.14729
 - **Pdf link:** https://arxiv.org/pdf/2307.14729
 - **Abstract**
 To ensure the reliable use of classification systems in medical applications, it is crucial to prevent silent failures. This can be achieved by either designing classifiers that are robust enough to avoid failures in the first place, or by detecting remaining failures using confidence scoring functions (CSFs). A predominant source of failures in image classification is distribution shifts between training data and deployment data. To understand the current state of silent failure prevention in medical imaging, we conduct the first comprehensive analysis comparing various CSFs in four biomedical tasks and a diverse range of distribution shifts. Based on the result that none of the benchmarked CSFs can reliably prevent silent failures, we conclude that a deeper understanding of the root causes of failures in the data is required. To facilitate this, we introduce SF-Visuals, an interactive analysis tool that uses latent space clustering to visualize shifts and failures. On the basis of various examples, we demonstrate how this tool can help researchers gain insight into the requirements for safe application of classification systems in the medical domain. The open-source benchmark and tool are at: https://github.com/IML-DKFZ/sf-visuals.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Mitigating Cross-Database Differences for Learning Unified HRTF  Representation
 - **Authors:** Yutong Wen, You Zhang, Zhiyao Duan
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2307.14547
 - **Pdf link:** https://arxiv.org/pdf/2307.14547
 - **Abstract**
 Individualized head-related transfer functions (HRTFs) are crucial for accurate sound positioning in virtual auditory displays. As the acoustic measurement of HRTFs is resource-intensive, predicting individualized HRTFs using machine learning models is a promising approach at scale. Training such models require a unified HRTF representation across multiple databases to utilize their respectively limited samples. However, in addition to differences on the spatial sampling locations, recent studies have shown that, even for the common location, HRTFs across databases manifest consistent differences that make it trivial to tell which databases they come from. This poses a significant challenge for learning a unified HRTF representation across databases. In this work, we first identify the possible causes of these cross-database differences, attributing them to variations in the measurement setup. Then, we propose a novel approach to normalize the frequency responses of HRTFs across databases. We show that HRTFs from different databases cannot be classified by their database after normalization. We further show that these normalized HRTFs can be used to learn a more unified HRTF representation across databases than the prior art. We believe that this normalization approach paves the road to many data-intensive tasks on HRTF modeling.
### Physics Informed Neural Network for Head-Related Transfer Function  Upsampling
 - **Authors:** Fei Ma, Thushara D. Abhayapala, Prasanga N. Samarasinghe, Xingyu Chen
 - **Subjects:** Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2307.14650
 - **Pdf link:** https://arxiv.org/pdf/2307.14650
 - **Abstract**
 Head-related transfer functions (HRTFs) capture the spatial and spectral features that a person uses to localize sound sources in space and thus are vital for creating an authentic virtual acoustic experience. However, practical HRTF measurement systems can only provide an incomplete measurement of a person's HRTFs, and this necessitates HRTF upsampling. This paper proposes a physics-informed neural network (PINN) method for HRTF upsampling. Unlike other upsampling methods which are based on the measured HRTFs only, the PINN method exploits the Helmholtz equation as additional information for constraining the upsampling process. This helps the PINN method to generate physically amiable upsamplings which generalize beyond the measured HRTFs. Furthermore, the width and the depth of the PINN are set according to the dimensionality of HRTFs under spherical harmonic (SH) decomposition and the Helmholtz equation. This makes the PINN have an appropriate level of expressiveness and thus does not suffer from under-fitting and over-fitting problems. Numerical experiments confirm the superior performance of the PINN method for HRTF upsampling in both interpolation and extrapolation scenarios over several datasets in comparison with the SH methods.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Unsupervised reconstruction of accelerated cardiac cine MRI using Neural  Fields
 - **Authors:** Tabita Catalán, Matías Courdurier, Axel Osses, René Botnar, Francisco Sahli Costabal, Claudia Prieto
 - **Subjects:** Image and Video Processing (eess.IV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2307.14363
 - **Pdf link:** https://arxiv.org/pdf/2307.14363
 - **Abstract**
 Cardiac cine MRI is the gold standard for cardiac functional assessment, but the inherently slow acquisition process creates the necessity of reconstruction approaches for accelerated undersampled acquisitions. Several regularization approaches that exploit spatial-temporal redundancy have been proposed to reconstruct undersampled cardiac cine MRI. More recently, methods based on supervised deep learning have been also proposed to further accelerate acquisition and reconstruction. However, these techniques rely on usually large dataset for training, which are not always available. In this work, we propose an unsupervised approach based on implicit neural field representations for cardiac cine MRI (so called NF-cMRI). The proposed method was evaluated in in-vivo undersampled golden-angle radial multi-coil acquisitions for undersampling factors of 26x and 52x, achieving good image quality, and comparable spatial and improved temporal depiction than a state-of-the-art reconstruction technique.
### Phenotype-preserving metric design for high-content image reconstruction  by generative inpainting
 - **Authors:** Vaibhav Sharma, Artur Yakimovich
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Quantitative Methods (q-bio.QM)
 - **Arxiv link:** https://arxiv.org/abs/2307.14436
 - **Pdf link:** https://arxiv.org/pdf/2307.14436
 - **Abstract**
 In the past decades, automated high-content microscopy demonstrated its ability to deliver large quantities of image-based data powering the versatility of phenotypic drug screening and systems biology applications. However, as the sizes of image-based datasets grew, it became infeasible for humans to control, avoid and overcome the presence of imaging and sample preparation artefacts in the images. While novel techniques like machine learning and deep learning may address these shortcomings through generative image inpainting, when applied to sensitive research data this may come at the cost of undesired image manipulation. Undesired manipulation may be caused by phenomena such as neural hallucinations, to which some artificial neural networks are prone. To address this, here we evaluate the state-of-the-art inpainting methods for image restoration in a high-content fluorescence microscopy dataset of cultured cells with labelled nuclei. We show that architectures like DeepFill V2 and Edge Connect can faithfully restore microscopy images upon fine-tuning with relatively little data. Our results demonstrate that the area of the region to be restored is of higher importance than shape. Furthermore, to control for the quality of restoration, we propose a novel phenotype-preserving metric design strategy. In this strategy, the size and count of the restored biological phenotypes like cell nuclei are quantified to penalise undesirable manipulation. We argue that the design principles of our approach may also generalise to other applications.
### A GPS Alternative using Electrical Transmission Grids as Precision  Timing Networks
 - **Authors:** Stephen Robson
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.14495
 - **Pdf link:** https://arxiv.org/pdf/2307.14495
 - **Abstract**
 It is widely recognised that over-reliance on GNSS (e.g GPS) for time synchronisation represents an acute threat to modern society, and a diversity of alternatives are required to mitigate the threat of an outage. This paper proposes a GNSS alternative using time dissemination over national scale transmission or distribution networks. The method utilises the same frequency bandwidth and coupling technology as established power line carrier technology in conjunction with modern chirp Spread Spectrum modulation. The basis of the method is the transmission of a time synchronised chirp from a central substation. During GNSS operation, all substations can estimate the time of flight by correlating the received chirp with a time-synchronised local copy. During GNSS outage, time sychronisation to the central substation is maintained by correcting for the precalculated time of flight. It is shown that recent advances in chirp spread spectrum allow for a computationally efficient algorithm with the capacity to compute hundreds of thousand of chirp correlations every second, improving the resolution of the system. ATP-EMTP simulations of the method on large transmission networks demonstrate sub-$\mu$s timing accuracy even in the presence of low SNR and impulsive noise. An FPGA based prototype is developed and experimental testing also demonstrates sub-$\mu$s accuracy for time dissemination over a distance of 700 m. Averaging over time allows operation down to -20 dB, which could extend the range of the system to a national scale.
### Mitigating Cross-Database Differences for Learning Unified HRTF  Representation
 - **Authors:** Yutong Wen, You Zhang, Zhiyao Duan
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2307.14547
 - **Pdf link:** https://arxiv.org/pdf/2307.14547
 - **Abstract**
 Individualized head-related transfer functions (HRTFs) are crucial for accurate sound positioning in virtual auditory displays. As the acoustic measurement of HRTFs is resource-intensive, predicting individualized HRTFs using machine learning models is a promising approach at scale. Training such models require a unified HRTF representation across multiple databases to utilize their respectively limited samples. However, in addition to differences on the spatial sampling locations, recent studies have shown that, even for the common location, HRTFs across databases manifest consistent differences that make it trivial to tell which databases they come from. This poses a significant challenge for learning a unified HRTF representation across databases. In this work, we first identify the possible causes of these cross-database differences, attributing them to variations in the measurement setup. Then, we propose a novel approach to normalize the frequency responses of HRTFs across databases. We show that HRTFs from different databases cannot be classified by their database after normalization. We further show that these normalized HRTFs can be used to learn a more unified HRTF representation across databases than the prior art. We believe that this normalization approach paves the road to many data-intensive tasks on HRTF modeling.
### Physics Informed Neural Network for Head-Related Transfer Function  Upsampling
 - **Authors:** Fei Ma, Thushara D. Abhayapala, Prasanga N. Samarasinghe, Xingyu Chen
 - **Subjects:** Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2307.14650
 - **Pdf link:** https://arxiv.org/pdf/2307.14650
 - **Abstract**
 Head-related transfer functions (HRTFs) capture the spatial and spectral features that a person uses to localize sound sources in space and thus are vital for creating an authentic virtual acoustic experience. However, practical HRTF measurement systems can only provide an incomplete measurement of a person's HRTFs, and this necessitates HRTF upsampling. This paper proposes a physics-informed neural network (PINN) method for HRTF upsampling. Unlike other upsampling methods which are based on the measured HRTFs only, the PINN method exploits the Helmholtz equation as additional information for constraining the upsampling process. This helps the PINN method to generate physically amiable upsamplings which generalize beyond the measured HRTFs. Furthermore, the width and the depth of the PINN are set according to the dimensionality of HRTFs under spherical harmonic (SH) decomposition and the Helmholtz equation. This makes the PINN have an appropriate level of expressiveness and thus does not suffer from under-fitting and over-fitting problems. Numerical experiments confirm the superior performance of the PINN method for HRTF upsampling in both interpolation and extrapolation scenarios over several datasets in comparison with the SH methods.
### Signal processing after quadratic random sketching with optical units
 - **Authors:** Rémi Delogne, Vincent Schellekens, Laurent Daudet, Laurent Jacques
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.14672
 - **Pdf link:** https://arxiv.org/pdf/2307.14672
 - **Abstract**
 Random data sketching (or projection) is now a classical technique enabling, for instance, approximate numerical linear algebra and machine learning algorithms with reduced computational complexity and memory. In this context, the possibility of performing data processing (such as pattern detection or classification) directly in the sketched domain without accessing the original data was previously achieved for linear random sketching methods and compressive sensing. In this work, we show how to estimate simple signal processing tasks (such as deducing local variations in a image) directly using random quadratic projections achieved by an optical processing unit. The same approach allows for naive data classification methods directly operated in the sketched domain. We report several experiments confirming the power of our approach.
### Audio Inputs for Active Speaker Detection and Localization via  Microphone Array
 - **Authors:** Davide Berghi, Philip J. B. Jackson
 - **Subjects:** Audio and Speech Processing (eess.AS); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.14739
 - **Pdf link:** https://arxiv.org/pdf/2307.14739
 - **Abstract**
 This study considers the problem of detecting and locating an active talker's horizontal position from multichannel audio captured by a microphone array. We refer to this as active speaker detection and localization (ASDL). Our goal was to investigate the performance of spatial acoustic features extracted from the multichannel audio as the input of a convolutional recurrent neural network (CRNN), in relation to the number of channels employed and additive noise. To this end, experiments were conducted to compare the generalized cross-correlation with phase transform (GCC-PHAT), the spatial cue-augmented log-spectrogram (SALSA) features, and a recently-proposed beamforming method, evaluating their robustness to various noise intensities. The array aperture and sampling density were tested by taking subsets from the 16-microphone array. Results and tests of statistical significance demonstrate the microphones' contribution to performance on the TragicTalkers dataset, which offers opportunities to investigate audio-visual approaches in the future.
### Stochastic Flight Plan Optimization
 - **Authors:** Ítalo Romani de Oliveira, Steve Altus, Sergey Tiourine, Euclides C. Pinto Neto, Alexandre Leite, Felipe C. F. de Azevedo
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2307.14883
 - **Pdf link:** https://arxiv.org/pdf/2307.14883
 - **Abstract**
 Airline operations are subject to many uncertainties, such as weather, varying demand, maintenance events, congestion, etc. Large amounts of information are currently ignored due to difficulties in processing big data sets. We explore the use of ensemble weather forecast, which presents several distinct weather predictions for the same time horizon. So far, ensemble forecasts have been very little exploited for flight planning purposes. Currently, airlines already carry out lots of statistical analyses on past data, and devise effective policies for how much fuel and payload an aircraft should carry and how much of time buffer should be used in the schedule. But these buffers can be further reduced by doing forward-looking stochastic optimization. The use of ensemble forecast allows to select a trajectory that optimizes the expected outcome of a flight for an array of scenarios, instead of optimizing for a single one. Besides, aircraft payload is another considerable source of uncertainty. We tested stochastic optimization, first with the objective of optimizing single flights, then with the objective of optimizing whole schedules. In one of the experiments, it was observed that, in 55.8% of the cases, stochastic optimization outperforms conventional optimization in terms of fuel consumption; in only 0.4% of the cases, conventional optimization wins; and, in the remaining 43.8% of the cases, they achieve equal results. The experiments with stochastic payload demonstrated that the use of payload uncertainty can squeeze a bit more fuel savings from the flight plan outcomes. But the use of this technology is not driven only by reducing overall fuel consumption. One optimization criterion can be the minimization of diversions or fuel emergencies, that is, choosing the candidate that minimizes the maximum fuel consumption (minimax).
### Weakly Supervised AI for Efficient Analysis of 3D Pathology Samples
 - **Authors:** Andrew H. Song, Mane Williams, Drew F.K. Williamson, Guillaume Jaume, Andrew Zhang, Bowen Chen, Robert Serafin, Jonathan T.C. Liu, Alex Baras, Anil V. Parwani, Faisal Mahmood
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Quantitative Methods (q-bio.QM)
 - **Arxiv link:** https://arxiv.org/abs/2307.14907
 - **Pdf link:** https://arxiv.org/pdf/2307.14907
 - **Abstract**
 Human tissue and its constituent cells form a microenvironment that is fundamentally three-dimensional (3D). However, the standard-of-care in pathologic diagnosis involves selecting a few two-dimensional (2D) sections for microscopic evaluation, risking sampling bias and misdiagnosis. Diverse methods for capturing 3D tissue morphologies have been developed, but they have yet had little translation to clinical practice; manual and computational evaluations of such large 3D data have so far been impractical and/or unable to provide patient-level clinical insights. Here we present Modality-Agnostic Multiple instance learning for volumetric Block Analysis (MAMBA), a deep-learning-based platform for processing 3D tissue images from diverse imaging modalities and predicting patient outcomes. Archived prostate cancer specimens were imaged with open-top light-sheet microscopy or microcomputed tomography and the resulting 3D datasets were used to train risk-stratification networks based on 5-year biochemical recurrence outcomes via MAMBA. With the 3D block-based approach, MAMBA achieves an area under the receiver operating characteristic curve (AUC) of 0.86 and 0.74, superior to 2D traditional single-slice-based prognostication (AUC of 0.79 and 0.57), suggesting superior prognostication with 3D morphological features. Further analyses reveal that the incorporation of greater tissue volume improves prognostic performance and mitigates risk prediction variability from sampling bias, suggesting the value of capturing larger extents of heterogeneous 3D morphology. With the rapid growth and adoption of 3D spatial biology and pathology techniques by researchers and clinicians, MAMBA provides a general and efficient framework for 3D weakly supervised learning for clinical decision support and can help to reveal novel 3D morphological biomarkers for prognosis and therapeutic response.
